incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -344,6 +345,57 @@ It means that this namespace is orphaned and its parent has been deleted. To fix
 $ kubectl hns set --root <namespace>
 ```
 
+<a name="use-limit-propagation"/>
+
+### Limit the propagation of an object to descendant namespaces
+
+To limit the propagation of an object, you can use any of the following exceptions selectors:
+
+```
+propagate.hnc.x-k8s.io/select 
+```
+The value for this selector has to be a valid k8s label selector. The object will only be propagated to namespaces whose labels match the label selector.
+
+```
+propagate.hnc.x-k8s.io/treeSelect 
+```
+With treeSelect, you can use a single namespace name to represent where this object should be propagated, or use a comma-separated list of negated (“!”) namespaces to represent which namespaces not to propagate to (e.g. `!child1, !child2` means do not propagate to `child1` and `child2`). For example, this can be used to propagate an object to a child namespace, but not a grand-child namespace, using the value `child, !grand-child`. 
+
+```
+propagate.hnc.x-k8s.io/none
+```
+Setting `none` to `true` (case insensitive) will result in the object not propagating to any descendant namespace. Any other value will be rejected.
```

Comment:
Does `true` need to be surrounded by double quotes (e.g. `"true"`)?

Reply:
I don't think so